crates/astria-core/src/protocol/transaction/v1alpha1/mod.rs

```diff
@@ -609,7 +610,7 @@ mod test {
             transaction_bytes: unsigned.to_raw().encode_to_vec().into(),
         };
 
-        insta::assert_json_snapshot!(tx.sha256_of_proto_encoding());
+        insta::assert_json_snapshot!(tx.id().as_bytes());
```

Comment:
Don't assert on the raw bytes, assert on the json representation of the identifier which will make this much nicer:
```suggestion
        insta::assert_json_snapshot!(tx.id());
```